app/my/page.tsx

```diff
@@ -0,0 +1,19 @@
+import { RowCard } from "@/app/components/ui";
+import Link from "next/link";
+
+export default function MyPage() {
```

Comment:
"My" is a bit vague, my-bets would be better. Other option is to call it portfolio.

Reply:
ok, we can change it to my bets

Reply:
We will show only bets there, so I think my-bets works fine for now. Or are you going to show more things under `/my/`?

Reply:
just bets